modules/apps/document-library/document-library-web/src/main/resources/META-INF/resources/document_library/css/_image_editor.scss

```diff
@@ -0,0 +1,31 @@
+.image-editor-modal {
+	.modal-body {
+		background-color: $dark;
+		padding: 0;
+	}
+
+	.tbar {
```

Comment:
Is this `.tbar` style always apply inside a modal? 
Maybe `.tbar` fit better outside of `.image-editor-modal` and make `.image-editor` or whatever without modal.

Reply:
At this point, yes. What you suggest probably will be needed once we implement it in the `item-selector`, but for now this is correct.